integration/k8s/README.md

```diff
@@ -0,0 +1,329 @@
+```
+
+Alternatively, you can also run `./scripts/installCC.sh` inside the cli_orgX_pod_name.
+
+#### Start chaincode container
+
+Now, we have installed our FPC Enclave Registry and the FPC Chaincode on each peer. Since we are using the chaincode
+as s server deployment model, it is now time to start the chaincodes.
+
+Run the following commands from your host terminal.
+
+```bash
+kubectl create configmap chaincode-config --from-env-file=packages/chaincode-config.properties -n hyperledger --dry-run=client -o yaml | kubectl apply -f -
+kubectl create -f chaincode/ercc/
+kubectl create -f chaincode/fpccc/
+```
+
+Alternatively, you can also call `just chaincode`.
+
+#### Approve
+
+Next, we complete chaincode installation process by approving and committing the chaincode definitions.
+
+For each Org:
+```bash
+# copy cli_orgX pod name from above and enter container
+kubectl exec -it cli_orgX_pod_name bash -n hyperledge
+
+# make sure you still have ERCC_PKG_ID, FPCCC_PKG_ID, and FPC_MRENCLAVE defined in your env (see above)
+
+# approve enclave registry
+peer lifecycle chaincode approveformyorg --channelID mychannel --name ercc --version 1.0 --package-id $ERCC_PKG_ID --sequence 1 -o orderer0:7050 --tls --cafile $ORDERER_CA
+
+# approve FPC chaincode
+peer lifecycle chaincode approveformyorg --channelID mychannel --name fpccc --version $FPC_MRENCLAVE --package-id $FPCCC_PKG_ID --sequence 1 -o orderer0:7050 --tls --cafile $ORDERER_CA
+```
+
+Alternatively, you can also run `./scripts/approve.sh` inside the cli_orgX_pod_name.
+
+#### Commit
+
+Pick some org that proceeds with committing chaincode.
+
+```bash
+# commit enclave registry
+peer lifecycle chaincode checkcommitreadiness --channelID mychannel --name ercc --version 1.0 --sequence 1 -o orderer0:7050 --tls --cafile $ORDERER_CA
+peer lifecycle chaincode commit -o orderer0:7050 --channelID mychannel --name ercc --version 1.0 --sequence 1 --tls true --cafile $ORDERER_CA --peerAddresses peer0-org1:7051 --tlsRootCertFiles /opt/gopath/src/github.com/hyperledger/fabric/peer/crypto/peerOrganizations/org1/peers/peer0-org1/tls/ca.crt --peerAddresses peer0-org2:7051 --tlsRootCertFiles /opt/gopath/src/github.com/hyperledger/fabric/peer/crypto/peerOrganizations/org2/peers/peer0-org2/tls/ca.crt
+
+# commit FPC chaincode
+peer lifecycle chaincode checkcommitreadiness --channelID mychannel --name fpccc --version  $FPC_MRENCLAVE --sequence 1 -o orderer0:7050 --tls --cafile $ORDERER_CA
+peer lifecycle chaincode commit -o orderer0:7050 --channelID mychannel --name fpccc --version  $FPC_MRENCLAVE --sequence 1 --tls true --cafile $ORDERER_CA --peerAddresses peer0-org1:7051 --tlsRootCertFiles /opt/gopath/src/github.com/hyperledger/fabric/peer/crypto/peerOrganizations/org1/peers/peer0-org1/tls/ca.crt --peerAddresses peer0-org2:7051 --tlsRootCertFiles /opt/gopath/src/github.com/hyperledger/fabric/peer/crypto/peerOrganizations/org2/peers/peer0-org2/tls/ca.crt
+```
+
+### Run our hello world example
```

Comment:
I notice that you switched to the auction chaincode, however, the demo still suggests to (i) compile and (ii) run the helloworld example. I guess these should be updated as well.